### PR TITLE
openstack: TeuthologyOpenStack.ssh continuous output

### PR DESCRIPTION
**Pull request by dachary**
Instead of waiting for the completion of the ssh command,
display stdout/stderr continously.

Signed-off-by: Loic Dachary <loic@dachary.org>

### CODE REVIEW

File: teuthology/openstack/__init__.py [hunk 1]
```diff
@@ -391,24 +389,27 @@ def cloud_init_wait(self):
...
                     log.exception('cloud_init_wait ' + ip)
                     raise
-                log.debug('cloud_init_wait ' + all_done)
+                log.debug('cloud_init_wait ' + tail)
                 try:
-                    stdin, stdout, stderr = client.exec_command(all_done)
-                    stdout.channel.settimeout(5)
-                    out = stdout.read()
-                    log.debug('cloud_init_wait stdout ' + all_done + ' ' + out)
+                    transport = client.get_transport()
```
Comment (zmc) [inline teuthology/openstack/__init__.py]: Could you add some comments to explain what's going on below please?

Comment (dachary) [inline teuthology/openstack/__init__.py]: it's  a series of call to get access to the I/O channel which can be used to display things line by line

File: teuthology/openstack/__init__.py [hunk 2]
```diff
@@ -523,17 +533,16 @@ def ssh(self, command):
...
-        out = ''
-        try:
-            out = stdout.read()
-            log.debug('ssh stdout ' + command + ' ' + out)
-        except Exception:
-            log.exception('ssh ' + command + ' failed')
-        err = stderr.read()
-        log.debug('ssh stderr ' + command + ' ' + err)
-        return out + ' ' + err
+        transport = client.get_transport()
```
Comment (zmc) [inline teuthology/openstack/__init__.py]: Could use a comment here as well explaining what the paramiko calls do

**Comment by dachary** [comment]
@zmc updated with a comment reminding what the purpose of the chain of paramiko chain of command is about

**Comment by zmc** [comment]
Thanks @dachary!

**Pull request merged**